File: Python/problem2.py
def willAttendanceAward(attendanceRecord):
    attendanceRecord = str(attendanceRecord)
    countA = attendanceRecord.count("A")
    isThreeOrMoreLate = "LLL" in attendanceRecord
    if(countA < 2 and not(isThreeOrMoreLate)):
        return True
    else:
        return False
    
if __name__ == "__main__":
    print(willAttendanceAward("PPPLALL"))
    

File: Java/problem2.java
package Java;

class Problem2 {
    
    public static int charCount(String str, char subString)
    {
        int count = 0;
        for(int i = 0; i < str.length(); i++)
        {
            if (str.charAt(i) == subString)
                count++;
        }
        return count;
    }

    public static boolean willAttendanceAward(String attendanceRecord)
    {
        int countA = charCount(attendanceRecord.toUpperCase(), 'A');
        boolean isThreeOrMoreLate = attendanceRecord.toUpperCase().contains("LLL");
        if(countA < 2 && !(isThreeOrMoreLate))
            return true;
        else
            return false;
    }
    
    public static void main(String[] args) {
        String s = "PPALPLLLLL";
        System.out.println(willAttendanceAward(s));
    }
}